overlord/snapstate/backend/mountunit.go

```diff
@@ -44,3 +49,7 @@ func removeMountUnit(mountDir string, meter progress.Meter) error {
 	sysd := systemd.New(systemd.SystemMode, meter)
 	return sysd.RemoveMountUnitFile(mountDir)
 }
+
+func (b Backend) RemoveSnapMountUnits(s snap.PlaceInfo, meter progress.Meter) error {
+	return wrappersRemoveMountUnitFiles(s, "", meter)
```

Comment:
relatedly this is mocked in the tests here and the code in wrappers doesn't have unit tests. Anyway as I said maybe the code in wrappers should just move herere because the other functions here use systemd directly anyway...

Reply:
Removed. For adding more tests, I'd like to get https://github.com/snapcore/snapd/pull/10704 merged first. Or if you are fine with the current amount of tests, of course we can land this first and add tests later. :-)